### PR TITLE
feat: Added PR alert if requesting sensitive info

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

Added notification to proof request if the proof is requesting sensitive info.
Sensitive info is defined by the `flaggedAttributes` in the oca bundle
Screenshot 2023-09-14 at 11 04 34 AM

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by bryce-mcmath — APPROVED** [review]

File: packages/legacy/core/App/screens/ProofRequest.tsx [hunk 1]
```diff
@@ -255,6 +257,22 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
     ...retrievedCredentials?.predicates,
   })
 
+  useEffect(() => {
+    // get oca bundle to see if we're presenting personally identifiable elements
+    proofItems.some(async (item) => {
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/screens/ProofRequest.tsx:262]: Missed this on my first readthrough, what is the purpose of the first `some` here? It doesn't seem to be used anywhere

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]

File: packages/legacy/core/App/screens/ProofRequest.tsx (hunk 1 above)
Comment (bryce-mcmath) [reply packages/legacy/core/App/screens/ProofRequest.tsx:262]: Could probably just be a `forEach`

Comment (wadeking98) [reply packages/legacy/core/App/screens/ProofRequest.tsx:262]: I'm checking to see if any of the proof items contains a flagged attribute. Some exits after the first success so it's a bit faster/more relevant than a foreach

Comment (bryce-mcmath) [reply packages/legacy/core/App/screens/ProofRequest.tsx:262]: Oh gotcha. Nice one, approved

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**